core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -664,7 +679,9 @@ export class WorkflowActionService {
       this.getTexeraGraph().getCommentBoxEditCommentStream(),
       this.getTexeraGraph().getCachedOperatorsChangedStream(),
       this.getTexeraGraph().getOperatorDisplayNameChangedStream(),
-      this.getTexeraGraph().getOperatorVersionChangedStream()
+      this.getTexeraGraph().getOperatorVersionChangedStream(),
+      this.getTexeraGraph().getPortDisplayNameChangedSubject(),
+      this.getTexeraGraph().getPortPropertyChangedStream()
```

Comment:
there might be some legacy reasons that we treat operator name and property change as two streams. however, for ports, since the display name is one of the properties of a port, is it essential to handle them separately?

Reply:
The separation actually is needed, since not all operator ports will have editable properties, but we want the name editing to be editable for each port.